examples/experimental/dagster-airlift/dagster_airlift/core/load_defs.py

```diff
@@ -53,26 +48,33 @@ def defs_from_state(
 
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines +44 to +47]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Actually I think we should fully decompose this and have _peer_ and _enrich_/_observe_ be separate loaders, with the _enrich_ step requiring the peered def in order to access the underlying cached data.<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/24910?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->